package.json

```diff
@@ -82,7 +80,6 @@
     "eslint-plugin-prettier": "^3.1.2",
     "eslint-plugin-react": "^7.11.0",
     "eslint-plugin-security": "^1.3.0",
-    "extract-text-webpack-plugin": "^4.0.0-beta.0",
```

Comment:
No longer recommended when using Webpack4+, which we do here. The `mini-css-extract-plugin` is the recommended approach as per Webpack docs and it is already a dep.